apps/foundation-demo/src/app/cart/components/cart-wrapper/cart-wrapper.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="cart-wrapper">
+  <div class="cart-wrapper__wrapper">
```

Comment:
I think this should be named this `cart-wrapper__main` or something. Goes better with `cart-wrapper__side-pane`

Reply:
done